gaea/tsconfig.json

```diff
@@ -0,0 +1,19 @@
+{
+    "compilerOptions": {
+        "module": "commonjs",
+        "target": "es6",
+        "noImplicitAny": true,
```

Comment:
Sorry, didn't update new-package script (#13548). This needs `"noImplicitThis": true` here.